lls/main.c

```diff
@@ -308,24 +308,13 @@ process_pkts(struct lls_config *lls_conf, struct gatekeeper_if *iface,
 			/* ARP reply was sent, so no free is needed. */
 			num_tx++;
 			continue;
-		case ETHER_TYPE_IPv6:
-			if (iface == &lls_conf->net->back) {
-				/*
-				 * Back interface can also see ND packets
-				 * received here.
-				 *
-				 * TODO Move RSS on the back interface
-				 * to a different block and pass ND
-				 * packets to the LLS using DPDK
-				 * packet classification and distribution.
-				 * Then, handle any non-ARP and non-ND
-				 * packets on the back interface. For now,
-				 * just drop them.
-				 */
-				add_pkt_ipv6_acl(&acl, bufs[i]);
```

Comment:
What about dropping the definition of `acl` above and the call to `process_pkts_ipv6_acl()` below?